test/SlowTests/Bugs/ConnectionStringParsing.cs

```diff
@@ -39,5 +39,26 @@ public void EnsureWellFormedConnectionStrings_Parsing_FailWithUnknownParameter()
             var dbParser = ConnectionStringParser<RavenConnectionStringOptions>.FromConnectionString("memory=true");
             Assert.Throws<ArgumentException>(() => dbParser.Parse());
         }
+
+        [Fact]
+        public void EnsureWellFormedConnectionStrings_ParsingMultiUrls_Successful()
+        {
+            var connectionStringParser = ConnectionStringParser<RavenConnectionStringOptions>.FromConnectionString("Urls=http://localhost:10301,http://localhost:10302,http://localhost:10303;");
```

Comment:
I thought the format was supposed to be:

`url=http://localhost:8080; url=http://localhost:8888`, etc, no?